### ISSUE TITLE
Markdown Support

### ISSUE THREAD

**Issue by dsivasuthan** [ostoaleonardo/notes#12]
I know there are other apps that can do this but I really like how this app looks with monospace font. 

This would require the files being saved to the local storage so other apps can edit the files as well.

**Comment by ostoaleonardo** [comment]
How should markdown integration work?

_Have the note text with the possibility to add additional markdown_ **or** _Replace the note text with markdown_

**Comment by rahulganes** [comment]
Replacing note text with markdown.

**Comment by dsivasuthan** [comment]
> Replacing note text with markdown.

Yeah, I would go about this route since Markdown can just text with plain text or get very complex with all the formatting. If the app has the ability to read an existing folder on the device or write to a folder, people can work around it.

**Comment by rahulganes** [comment]
A toggle to enable markdown will also be a good option too. If users don't like it, they can disable it then.
